content/ecosystem/distributions/distributions.toml

```diff
@@ -0,0 +1,67 @@
+[[distributions]]
+name = "openDesk"
+description = "The customisable office and collaboration suite designed specifically for your needs in public administration."
+vendor = "ZenDiS GmbH"
+maturity = "Stable"
+frameworks = ["Helmfile", "Helm", "Kubernetes"]
+licence = "Apache-2.0"
+repository = "https://gitlab.opencode.de/bmi/opendesk"
+support_level = "Commercial"
```

Comment:
Should we add a link to where people can get commercial support?

@AmandineLP @bluesomewhere should the Foundation list all commercial distributions, or only its members / partners?

If we list all, should we add a distinctive badge for the ones supporting the Foundation and rank them higher? Hide those who don't by default, and display them when the user clicks on "expand"?